ModelBundle/Repository/TemplateRepository.php

```diff
@@ -17,13 +17,31 @@ class TemplateRepository extends AbstractAggregateRepository implements FieldAut
     use AreaFinderTrait;
 
     /**
+     * @param $template TemplateInterface
+     * @param string    $areaId
+     *
+     * @return null|AreaInterface
+     */
+    public function findAreaInTemplateByAreaId(TemplateInterface $template, $areaId)
```

Comment:
@alavieille : I'm not sure this is the right place for this method: maybe it could be nice to have an AreaRepository. The template repository should be used to get template(s), not other things.